packages/loot-core/src/client/actions/queries.ts

```diff
@@ -249,18 +239,14 @@ export function updateGroup(group) {
   // time
   const { categories, ...rawGroup } = group;
 
-  return async dispatch => {
+  return async () => {
     await send('category-group-update', rawGroup);
-    await dispatch(getCategories());
   };
 }
 
 export function deleteGroup(id, transferId?) {
-  return async function (dispatch) {
+  return async function () {
     await send('category-group-delete', { id, transferId });
-    await dispatch(getCategories());
-    // See `deleteCategory` for why we need this
-    await dispatch(getPayees());
```

Comment:
Same comment above. I don't think we need to update payees on category delete. Please correct me if I'm wrong on this one.